pipeline/deploy.sh

```diff
@@ -102,33 +102,19 @@ echo "sv_kubeconfig_content=$(cat $SV_KUBECONFIG)" > ./sv_kubeconfig_content.yam
 # Pod status on testbed before patching the CSI Images
 kubectl get pods -n vmware-system-csi
 
-K8S_MAJOR_VERSION=$(kubectl version -o json | jq .serverVersion.major | tr -d '"')
-K8S_MINOR_VERSION=$(kubectl version -o json | jq .serverVersion.minor | tr -d '"')
-
-echo "K8S_MAJOR_VERSION=$K8S_MAJOR_VERSION" >> build.env
-echo "K8S_MINOR_VERSION=$K8S_MINOR_VERSION" >> build.env
-
-# Replace the kubernetes version in kustomization.yaml
-sed -i "s#- ../../manifests/supervisorcluster/1.22#- ../../manifests/supervisorcluster/$K8S_MAJOR_VERSION.$K8S_MINOR_VERSION#g" pipeline/dev/kustomization.yaml
-
-echo "cat pipeline/dev/kustomization.yaml"
-cat pipeline/dev/kustomization.yaml
- 
-if ! kustomize build pipeline/dev | envsubst | kubectl apply -f -;
-then
-	echo "Error patching the CSI images in the testbed."
-	exit 1
-fi
+kubectl set env deployment/vsphere-csi-controller -n vmware-system-csi --containers=vsphere-syncer FULL_SYNC_INTERVAL_MINUTES=2 VOLUME_HEALTH_INTERVAL_MINUTES=2
+kubectl set image deployment/vsphere-csi-controller -n vmware-system-csi vsphere-syncer="$VSPHERE_SYNCER_IMAGE" vsphere-csi-controller="$VSPHERE_CSI_CONTROLLER_IMAGE"
+kubectl set image deployment/vsphere-csi-webhook -n vmware-system-csi vsphere-webhook="$VSPHERE_SYNCER_IMAGE"
```

Comment:
While the `kubectl set image` command can be used to update the images, there are limitations in this approach on what other patches you could do. I would suggest using `kustomize` to patch the images as it is a more robust way of patching the k8s environments and is easy to extend with more patches/changes in the future. Any reason for not using `kustomize`?

Reply:
Created a task to address the review comment - further updates will be made on the PR.

Reply:
`kubectl set` will also restart the deployment each time it's used. Using `kustomize` will take care of patching required changes in one shot.

As indicated, this will be taken up in a subsequent MR.